docs/api/browser-window.md

```diff
@@ -864,7 +864,7 @@ Returns `Boolean` - Whether the window is in normal state (not maximized, not mi
 
 * `aspectRatio` Float - The aspect ratio to maintain for some portion of the
 content view.
-* `extraSize` [Size](structures/size.md) - The extra size not to be included while
+* `extraSize` [Size](structures/size.md) (Optional) - The extra size not to be included while
```

Comment:
any particular reason not to be consistent re: capitalization between `(Optional)` and `(optional)`?

Reply:
Fair, enforcing lower case now 👍